### PR TITLE
Perm fixes

### PR DESCRIPTION
**Pull request by luisza**
This PR has 2 big new functionalities:
1. Allows set **template_father**  so if someone don't want to use adminlte base template can overwrite easy the base template loaded.  The default value is template_father = "cruds/base.html"
2. Filter button and links  based in user permissions.

Also fixed inlines when 2 or more inlines are availble (modal didn't show)

### CODE REVIEW

**Comment by luisza** [comment]
New pip release could be great.

**Comment by oscarmlage** [comment]
New release done, _version 0.0.4_ is now on pypi. Thank you so much for your efforts.

File: cruds_adminlte/templates/cruds/create.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends "cruds/base.html" %}
+{% extendstemplate_father %}
```
Comment (matejv) [inline cruds_adminlte/templates/cruds/create.html]: A space is missing here

Comment (oscarmlage) [inline cruds_adminlte/templates/cruds/create.html]: Thank you @matejv, 4 eyes always are better than 2... and we need collaborators here ;)

Comment (oscarmlage) [inline cruds_adminlte/templates/cruds/create.html]: Fixed 4a64ca093df390e0371d42c1b1c453328f0fc276

**Pull request merged**